fadtk/model_loader.py

```diff
@@ -613,27 +713,28 @@ def _get_embedding(self, audio: np.ndarray) -> np.ndarray:
 
         return out
 
-
-
-def get_all_models() -> list[ModelLoader]:
+def get_all_models(audio_len=None) -> list[ModelLoader]:
```

Comment:
if adding the same docstring for `audio_len` is tedious, we can at least add it here.